src/utils/ranges.ts

```diff
@@ -135,6 +135,13 @@ function isTimeInRange({ start, end } : IRange, time : number) : boolean {
   return start <= time && time < end;
 }
 
+function isTimeInTimeRanges(timeRanges : TimeRanges, time : number) : boolean {
+  for (let i = 0; i < timeRanges.length; i++) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Refactor this loop; it's executed only once ([typescript:S1751](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS1751))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=318&issues=AWYbSRfPM63is4RDZrpD&open=AWYbSRfPM63is4RDZrpD)

[ISSUE_KEY]: AWYbSRfPM63is4RDZrpD